app/backend/application/database.py

```diff
@@ -20,11 +22,16 @@ def delete_record(record_id: str):
     db.session.commit()
 
 
-def set_record_uses(record_id: str, max_uses: int):
+def set_record_uses_expiry(record_id: str, max_uses: int, expiry_hours: int, expiry_minutes: int):
```

Comment:
Should be `max_uses: Optional[int]` since `None` is valid there.

Reply:
fixed in 16027ce